test/test_functional_tensor.py

```diff
@@ -747,10 +748,8 @@ def test_gaussian_blur(self):
 
                         for fn in [F.gaussian_blur, scripted_transform]:
                             out = fn(tensor, kernel_size=ksize, sigma=sigma)
-                            self.assertEqual(true_out.shape, out.shape, msg="{}, {}".format(ksize, sigma))
-                            self.assertLessEqual(
-                                torch.max(true_out.float() - out.float()),
-                                1.0,
+                            torch.testing.assert_close(
+                                out, true_out, rtol=0.0, atol=1.0, check_stride=False,
```

Comment:
we're comparing the max abs difference here instead of the max difference in master, but it's probably more correct with the proposed changes